backend/build.gradle

```diff
@@ -1,6 +1,12 @@
+buildscript {
```

Comment:
⚠️ changes in this file due to this [release note](https://github.com/liquibase/liquibase-gradle-plugin/blob/master/doc/releases.md#releases)

> IMPORTANT: As of version 3.0.0, this plugin no longer works with Liquibase versions prior to 4.24, and it requires liquibase to be in the buildscript classpath, in addition to the liquibaseRuntime dependency.